docs/design/ovs-pipeline.md

```diff
@@ -183,1015 +269,1363 @@ spec:
   appliedTo:
     - podSelector:
         matchLabels:
-          app: server
+          app: nginx
   ingress:
-    - action: Drop
+    - action: Allow
       from:
         - podSelector:
             matchLabels:
-              app: notClient
+              app: nginx
       ports:
         - protocol: TCP
           port: 80
+    - action: Drop
   egress:
     - action: Allow
       to:
         - podSelector:
             matchLabels:
-              app: dns
+              app: nginx
       ports:
-        - protocol: UDP
-          port: 53
+        - protocol: TCP
+          port: 80
+    - action: Reject
+```
+
+This ACNP is applied to all Pods with the `app: nginx` label in all Namespaces. For these Pods, it only allows TCP traffic
+on port 80 from all Pods with the label `app: nginx` and drop others. In addition to the ingress rules, this policy only
+allows egress TCP traffic on port 80 to all Pods with the label `app: nginx` and reject others. Similar to Kubernetes
+NetworkPolicy, Antrea will only install OVS flows for this policy on Nodes for which some of the Pods are the target of
+the policy. We still use `app:nginx` Pods mentioned above as the target of this policy.
```

Comment:
That may be a big change because you already captured the flows for these policies, but I wonder if we should use more meaningful policy examples for the documentation. It doesn't make too much sense to have an ACNP which does pretty much the same thing as the K8s NP, and it also doesn't make sense to have nginx Pods talking to other nginx Pods, as they are both "servers". That second one may be my fault because I probably wrote the original K8s NP example in this doc.
Maybe we can leave it as a future enhancement though, given that this PR has been open for a while.

Reply:
Make sense. I will update it as you said.